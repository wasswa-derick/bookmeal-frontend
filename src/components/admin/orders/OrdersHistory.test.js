import React from "react";
import { shallow } from "enzyme";
import { OrdersHistory } from "./OrdersHistory";

const err = {
  response: {
    status: 400
  }
};
const fn = () => Promise.reject(err);

describe("OrdersHistory", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<OrdersHistory getOrders={fn} orders={[]} />);
  });

  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
