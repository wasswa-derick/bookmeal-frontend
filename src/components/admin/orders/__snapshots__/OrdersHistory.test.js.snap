// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrdersHistory should render correctly 1`] = `
<div
  className="container"
>
  <div>
    <h4>
      Order History
    </h4>
    <hr />
  </div>
  <Component
    component="div"
    loaded={false}
    loadedClassName="loadedContent"
    parentClassName="loader"
  >
    <table
      className="table table-striped"
      id="dtable"
    >
      <thead>
        <tr
          role="row"
        >
          <th>
            Customer Name
          </th>
          <th>
            Customer Contact
          </th>
          <th>
            Order Count
          </th>
          <th>
            Meals Ordered
          </th>
          <th>
            Total Cost
          </th>
          <th>
            Date Ordered
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </Component>
</div>
`;
